pkg/cpuallocator/allocator.go

```diff
@@ -943,6 +984,40 @@ func (c *cpuPriorities) cmpCPUSet(csetA, csetB cpuset.CPUSet, prefer CPUPriority
 		return 0
 	}
 
+	// For low prio request, favor cpuset with the tightest fit.
+	if cpuCnt > 0 && prefer == PriorityLow {
+		prefA := csetA.Intersection(c[prefer]).Size()
+		prefB := csetB.Intersection(c[prefer]).Size()
+		if prefA >= cpuCnt && prefB >= cpuCnt {
+			return prefA - prefB
```

Comment:
In order to prefer the tightest fit, line 992 should prefer A (in other words return > 0) if prefA < prefB. But now `prefA - prefB` works the other way around: it returns > 0 when prefA > prefB. If I'm not mistaken, line 992 should be `return prefB - prefA` similartly to line 1028.